hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/V0500TokenSchema.java

```diff
@@ -39,6 +43,8 @@
 public class V0500TokenSchema extends Schema {
     private static final Logger log = LogManager.getLogger(V0500TokenSchema.class);
     private static final String SHARED_VALUES_KEY = "V0500_FIRST_STORAGE_KEYS";
+    private static final long MAX_PENDING_AIRDROPS = 1_000_000_000L;
```

Comment:
A question for the PR reviewer: do you think this is accurate?

Reply:
I am not sure what is the decided value for this. @Nana-EC @netopyr Do you know if this is correct ?